docs/eips/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Ethereum Improvement Proposals (EIPs)
+meta:
+  - name: description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+  - property: og:title
+    content: Ethereum Improvement Proposals (EIPs) | Ethereum.org
+  - property: og:description
+    content: The basic information you need to understand Ethereum Improvement Proposals (EIPs).
+lang: en-US
+sidebar: auto
+---
+
+# Introduction to Ethereum Improvement Proposals (EIPs)
+
+## What is an EIP?
+
+[Ethereum Improvement Proposals (EIPs)](https://eips.ethereum.org/) are standards specifying potential new features or processes for Ethereum. EIPs contain technical specifications for the proposed changes and act as the “source of truth” for the community. Network upgrades for Ethereum are discussed and developed through the EIP process.
```

Comment:
Perhaps we should add a few words about dapps to make it clear it's not only network upgrades that are discussed in EIPs

```suggestion
[Ethereum Improvement Proposals (EIPs)](https://eips.ethereum.org/) are standards specifying potential new features or processes for Ethereum. EIPs contain technical specifications for the proposed changes and act as the “source of truth” for the community. Network upgrades and application standards for Ethereum are discussed and developed through the EIP process.
```